aws-android-sdk-s3-test/src/androidTest/java/com/amazonaws/mobileconnectors/s3/transferutility/PauseTransferIntegrationTest.java

```diff
@@ -32,7 +32,7 @@
 
 public final class PauseTransferIntegrationTest extends S3IntegrationTestBase {
 
-    private static final String BUCKET_NAME = "amazon-transfer-util-integ-test-" + new Date().getTime();
+    private static final String BUCKET_NAME = "android-sdk-transfer-util-integ-test-" + new Date().getTime();
```

Comment:
In some of these constructions, we're using `Date().getTime()`, in others: `System.currentTimeMillis()` -- what do you think about being consistent?